### PR TITLE
Added tests for new functionality and improved older tests on RU tab

### PR DESCRIPTION
**Pull request by ChristopherMShelton**
# Motivation and Context
Added new functionality for respondents tab which required new suite of tests.

# What has changed
added tests for new buttons on respondent tab, made some changes to older tests from the RU tab to make them more accurate

# How to test?
run the tests

# Links
https://trello.com/c/JOPM1IFq/394-us216-view-respondent-details-m

### CODE REVIEW

File: acceptance_tests/features/steps/enrolment_status_disable.py [hunk 1]
```diff
@@ -9,6 +11,27 @@ def internal_user_view_ru_details(_):
...
+
+@then('the user is redirected to a confirmation screen')
+def confirm_verification_screen(_):
+    assert 'Edit collection exercise details' in browser.title
+
+
+@then('the user is redirected to the respondents page and a success message is displayed')
+def confirm_success_message(_):
+    assert 'Respondents | Survey Data Collection' in browser.title
+    assert 'Enrolment status changed' in browser.find_by_id('success').text
```
Comment (owencjones) [inline acceptance_tests/features/steps/enrolment_status_disable.py]: Are both these asserts necessary to confirm the step has been met?  Where possible, a test should assert a single thing - but that's not always possible.

File: acceptance_tests/features/edit_respondent_details.feature [hunk 2]
```diff
@@ -1,26 +1,28 @@
 @business
 @standalone
+@wip
```
Comment (mfcrocker) [inline acceptance_tests/features/edit_respondent_details.feature]: Should this still be `@wip`?

File: acceptance_tests/features/edit_respondent_details.feature [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
-    Then the respondent account details become editable
-
-  @us057-s02
-  @fixture.setup.data.with.enrolled.respondent.user.and.internal.user
-  Scenario: All fields are required to be populated
-    Given the internal user chooses to change account details
-    When they remove the old contact number
-    And they click save
-    Then the changes will not be saved and they are informed that all fields are required
+#  @us057-s01
```
Comment (mfcrocker) [inline acceptance_tests/features/edit_respondent_details.feature]: Can these tests be deleted?

File: acceptance_tests/features/edit_respondent_details.feature (hunk 2 above)
Comment (ChristopherMShelton) [reply acceptance_tests/features/edit_respondent_details.feature]: nope, removed

File: acceptance_tests/features/edit_respondent_details.feature (hunk 3 above)
Comment (ChristopherMShelton) [reply acceptance_tests/features/edit_respondent_details.feature]: no, but they should be uncommented 👍

File: acceptance_tests/features/steps/enrolment_status_disable.py (hunk 1 above)
Comment (ChristopherMShelton) [reply acceptance_tests/features/steps/enrolment_status_disable.py]: I think so, part of the test is making sure it doesn't redirect to the RU tab- and if it does the other assert would return true also

Comment (owencjones) [reply acceptance_tests/features/steps/enrolment_status_disable.py]: Fair play, consider this resolved

File: acceptance_tests/features/edit_respondent_details.feature (hunk 2 above)
Comment (owencjones) [reply acceptance_tests/features/edit_respondent_details.feature]: @mfcrocker Resolved?

File: acceptance_tests/features/steps/respondent_verification.py [hunk 4]
```diff
@@ -69,3 +70,40 @@ def resend_verification_page(_):
...
+    browser.find_by_id('cancel').click()
+
+
+@then('they are redirected to the respondents page and a confirmation message is displayed')
+def confirm_email_resend(_):
+    assert browser.find_by_text('Verification email re-sent')
+
+
+
+
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/respondent_verification.py]: Removed empty lines, we normally have one empty line at the end of a file 🎨

File: acceptance_tests/features/steps/edit_respondent_details.py [hunk 5]
```diff
@@ -20,6 +20,12 @@ def open_edit_details_change_email(context):
     reporting_unit.click_edit_details(context.short_name, context.respondent_email)
 
 
+# @given('the internal user has navigated to a respondents details page')
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/edit_respondent_details.py]: Why is this commented out?

File: acceptance_tests/features/edit_respondent_details.feature [hunk 6]
```diff
@@ -1,5 +1,6 @@
 @business
 @standalone
+
```
Comment (ryangrundy7) [inline acceptance_tests/features/edit_respondent_details.feature]: Remove this space 🎨

File: acceptance_tests/features/enrolment-status-disable-enable.feature [hunk 7]
```diff
@@ -38,4 +38,21 @@ Feature: Disable respondent enrolment status
...
+
+
+  @fixture.setup.data.with.enrolled.respondent.user.and.internal.user
+   Scenario: internal user clicks confirm button on confirmation screen
+     Given the internal user has navigated to a respondents details page
+     When the user clicks either disable or re-enable
+     And the user clicks the confirm button
+     Then the user is redirected to the respondents page and a success message is displayed
+
+
```
Comment (ryangrundy7) [inline acceptance_tests/features/enrolment-status-disable-enable.feature]: remove extra space 🎨

File: acceptance_tests/features/steps/respondent_unlocking_account.py [hunk 8]
```diff
@@ -55,3 +55,23 @@ def password_reset_complete(_):
 @when('they confirm their password reset')
 def submitting_password(_):
     forgotten_password_respondent.click_confirm_new_password()
+
+
+@when('they click the unlock button')
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/respondent_unlocking_account.py]: This needs to be a Given

**Pull request merged**